.github/workflows/ci.yml

```diff
@@ -162,7 +162,7 @@ jobs:
     - name: Prepare for the autotests
       working-directory: ./autotest
       run: |
-        pytest -v ci_prepare.py
+        get-modflow "$HOME/.local/bin"
```

Comment:
Use something like this:
https://github.com/modflowpy/flopy/blob/3e2b5fb0ec0fc4b58854fce6155eaa40c25603c6/.github/workflows/rtd.yml#L54-L56

Reply:
Thanks!